drag_and_drop_v2/public/js/view.js

```diff
@@ -115,7 +116,7 @@
                     },
                     style: {
                         top: zone.y_percent + '%', left: zone.x_percent + "%",
-                        width: zone.width_percent + '%', height: zone.height_percent + "%",
+                        width: zone.width_percent + '%', height: zone.height_percent + "%"
```

Comment:
@agaylard Nit: Can you bring the trailing comma back (to make future diffs nicer)? I know this isn't supported in some older browsers, but since we are only supporting ES5+ browsers, including it won't be a problem.